ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/daemon-terminattr.j2

```diff
@@ -1,12 +1,38 @@
-| gzip | {% for cvp_ip in daemon_terminattr.ingestgrpcurl.ips %}{{ cvp_ip }}:{{ daemon_terminattr.ingestgrpcurl.port }}{% if not loop.last %},{% endif %}{% endfor %} | {{ daemon_terminattr.ingestauth_key | arista.avd.default('UNSET') }} | {{ daemon_terminattr.smashexcludes }} | {{ daemon_terminattr.ingestexclude }} | {{ daemon_terminattr.ingestvrf }} | {{ daemon_terminattr.ingestvrf }} | {{ daemon_terminattr.disable_aaa | default (false) }} |
+| CV Compression | CloudVision Servers |  VRF | Authentication | Smash Excludes | Ingest Exclude | AAA Disabled |
+| -------------- | ------------------- | ---- | -------------- | -------------- | -------------- | ------------ |
+{% for cluster_name in daemon_terminattr.clusters | arista.avd.natural_sort %}{% set cluster = daemon_terminattr.clusters[cluster_name] %}
+{%     set url = cluster.cvaddrs | join(',') %}
+{%     if cluster.cvauth.method is arista.avd.defined('key') and cluster.cvauth.key is arista.avd.defined %}
+{%         set auth = 'key,' ~ cluster.cvauth.key %}
+{%     elif cluster.cvauth.method is arista.avd.defined('token') and cluster.cvauth.token_file is arista.avd.defined %}
+{%         set auth = 'token,' ~ cluster.cvauth.token_file %}
+{%     elif cluster.cvauth.method is arista.avd.defined('token-secure') and cluster.cvauth.token_file is arista.avd.defined %}
+{%         set auth = 'token-secure,' ~ cluster.cvauth.token_file %}+
+{%     endif%}
+{%     if cluster.cvvrf is arista.avd.defined %}
+{%         set cvvrf = cluster.cvvrf %}
+{%     else %}
+{%         set cvvrf = daemon_terminattr.cvvrf %}
+{%     endif %}
+| gzip | {{ url }} | {{ cvvrf }} | {{ auth }} | {{ daemon_terminattr.smashexcludes | arista.avd.default('-') }} | {{ daemon_terminattr.ingestexclude | arista.avd.default('-') }} | {{ daemon_terminattr.disable_aaa | arista.avd.default (false) }} |
+{% endfor %}
+{% if daemon_terminattr.cvaddrs is arista.avd.defined %}
+{%     set url = daemon_terminattr.cvaddrs | join(',') %}
+{% endif %}
+{% if daemon_terminattr.cvauth.method is arista.avd.defined('key') and daemon_terminattr.cvauth.key is arista.avd.defined %}
+{%     set auth = 'key,' ~ daemon_terminattr.cvauth.key %}
+{% elif daemon_terminattr.cvauth.method is arista.avd.defined('token') and daemon_terminattr.cvauth.token_file is arista.avd.defined %}
+{%     set auth = 'token,' ~ daemon_terminattr.cvauth.token_file %}
+{% elif daemon_terminattr.cvauth.method is arista.avd.defined('token-secure') and daemon_terminattr.cvauth.token_file is arista.avd.defined %}
+{%     set auth = 'token-secure,' ~ daemon_terminattr.cvauth.token_file %}
+{% endif %}
+| gzip | {{ url }} | {{ daemon_terminattr.cvvrf }} | {{ auth }} | {{ daemon_terminattr.smashexcludes | arista.avd.default('-') }} | {{ daemon_terminattr.ingestexclude | arista.avd.default('-') }} | {{ daemon_terminattr.disable_aaa | arista.avd.default (false) }} |
```

Comment:
`auth` field is not protected and has no default value in the `if/elif` block above. Please use `| arista.avd.default('-')`.

Also column name is `Disabled AAA` and its default value is `False` (disable_aaa) which is a bit misleading.